src/main/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractor.java

```diff
@@ -78,6 +78,12 @@ public DestinationExtractor(String matchingTableFile) {
     public void initialize() {
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/e2a78c5c2bf7b2142cfc89863c9ea3e7b960de24